hibernate-core/src/main/java/org/hibernate/SharedSessionContract.java

```diff
@@ -18,18 +18,15 @@
 import org.hibernate.jdbc.Work;
 import org.hibernate.procedure.ProcedureCall;
 import org.hibernate.query.QueryProducer;
-import org.hibernate.query.UnknownSqlResultSetMappingException;
 import org.hibernate.query.criteria.HibernateCriteriaBuilder;
 
 /**
- * Contract methods shared between {@link Session} and {@link StatelessSession}.
- * <p/>
- * NOTE : Poorly named.  "shared" simply indicates that its a unified contract between {@link Session} and
- * {@link StatelessSession}.
- * 
+ * Declares operations that are common between {@link Session} and {@link StatelessSession}.
+ *
  * @author Steve Ebersole
  */
-public interface SharedSessionContract extends QueryProducer, Serializable {
+//NOTE : This interface is poorly named: "shared" simply indicates that it's a unified contract
```

Comment:
Please remove this line.  We had a very long and drawn out discussion and consensus about the naming here.  Whether you like it or not, you had the chance to chime in when we had that original discussion ;)

Reply:
I didn't add that line, it was already there, I just moved it out of the Javadoc and into a regular comment.

Do you want me to put it back into the Javadoc? Or just delete it altogether (my preference).

Reply:
Just remove it